cmd/commands/destroy_cluster.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2023 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package command
+
+import (
+	"fmt"
+	"github.com/rook/kubectl-rook-ceph/pkg/crds"
+	"github.com/rook/kubectl-rook-ceph/pkg/logging"
+	"github.com/spf13/cobra"
+	"os"
+	"strings"
+)
+
+const (
+	destroyClusterQuestion = "Are you sure you want to destroy the cluster in namespace %q? If absolutely certain, enter: " + destroyClusterAnswer
+	destroyClusterAnswer   = "yes-really-destroy-cluster"
+)
+
+// DestroyClusterCmd represents the command for destroy cluster
+var DestroyClusterCmd = &cobra.Command{
+	Use:   "destroy-cluster",
+	Short: "delete all Rook CRDs",
+	Run: func(cmd *cobra.Command, args []string) {
+		question := fmt.Sprintf(destroyClusterQuestion, CephClusterNamespace)
+		confirmation, err := confirmPrompt(question, destroyClusterAnswer)
+		if err != nil {
+			logging.Fatal(err)
+		}
+
+		logging.Info(confirmation)
+		clientsets := GetClientsets(cmd.Context())
+		crds.DeleteCustomResources(cmd.Context(), clientsets, CephClusterNamespace)
+	},
+}
+
+func confirmPrompt(question, key string) (string, error) {
```

Comment:
#170 has a helper method for confirming a prompt. After that's merged, let's refactor to also use `PromptToContinueOrCancel()` here.

Reply:
sure, make sense

Reply:
#170 is merged, can you use its confirm prompt now?